lib/tmuxinator/deprecations.rb

```diff
@@ -1,19 +1,12 @@
 module Tmuxinator
   module Deprecations
-    def rvm?
-      yaml["rvm"]
-    end
-
-    def rbenv?
-      yaml["rbenv"]
-    end
-
-    def pre_tab?
-      yaml["pre_tab"]
-    end
-
-    def cli_args?
-      yaml["cli_args"]
+    def method_missing(name, *args, &block)
+      {:rvm? => "rvm", :rbenv? => "rbenv", :pre_tab? => "pre_tab", :cli_args? => "cli_args"}.each do |key,value|
```

Comment:
Line is too long. [112/80]<br>Space missing after comma.<br>Space inside { missing.<br>Space inside } missing.